app/Http/Controllers/EmployeeController.php

```diff
@@ -29,8 +29,10 @@ public function draw(Request $request){
         $start = $request->query('start', 0);
         $length = $request->query('length', 10);
         $totalEmployees =   Employee::count();
-        $arrayColumns = ['','company.name', 'company.branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'bankAccount.account_no'];
+        $arrayColumns = ['', 'company_name', 'branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'account_no'];
```

Comment:
remove this array and receive the column from client side. no point of defining an array

Reply:
Yes got it....made the change @Raza9798 #83